render/line.go

```diff
@@ -24,6 +24,45 @@ func NewThickLine(x1, y1, x2, y2 float64, c color.Color, thickness int) *Sprite
 	return NewSprite(minX-float64(thickness), minY-float64(thickness), rgba)
 }
 
+// DrawThickLine acts like DrawlineOnto, but takes in thickness of the given line
```

Comment:
We can correct that when we overhaul polygon drawing, to standardize thickness for polygon types